### PR TITLE
Jobs definitions rewrites

### PR DESCRIPTION
**Pull request by VoR0220**
- Cleaning up docs and adding moar purpose to our conquest of the sprawl.

### CODE REVIEW

File: docs/specs/index.md [hunk 1]
```diff
@@ -35,24 +29,18 @@ Service definition files are formatted using `toml`.
...
 
+Jobs are a portion of the upcoming Monax package management tooling. They enable ease of automation of smart contract deployment and runtime configuration with special utilizations for Burrow, the Monax blockchain. All tasks can be run through a simple yaml file and cover a range of cases from interaction with smart contracts, assertion testing, name registry, querying the state of the chain, setting permissions on certain addresses, or sending transactions. 
+
 Jobs are defined in **job definition files**.
 
-Action definition files are formatted in `yaml` and default file is `epm.yaml`.
+Job definition files are formatted in `yaml` and default file is `epm.yaml`.
 
-Examples of job definition files are available in the jobs_fixtures directory.
+Examples of monax job definition files are available in the [jobs_fixtures](/tests/jobs_fixtures) directory.
```
Comment (zramsay) [inline docs/specs/index.md]: needs to have url of repo because this goes on the website ...

File: docs/specs/jobs_specification.md [hunk 2]
```diff
@@ -12,28 +12,32 @@ title: "Specifications | Jobs Specification"
 	<em>Note: As of 2017, our product has been renamed from Eris to Monax. This documentation refers to an earlier version of the software prior to this name change (<= 0.16). Later versions of this documentation (=> 0.17) will change the <code>eris</code> command and <code>~/.eris</code> directory to <code>monax</code> and <code>~/.monax</code> respectively.</em>
 </div>
 
-Jobs are defined in **job definition files**.
+Jobs are a portion of the upcoming Monax package management tooling. They enable ease of automation of smart contract deployment and runtime configuration with special utilizations for Burrow, the Monax blockchain. All tasks can be run through a simple yaml file and cover a range of cases from interaction with smart contracts, assertion testing, name registry, querying the state of the chain, setting permissions on certain addresses, or sending transactions.
```
Comment (zramsay) [inline docs/specs/jobs_specification.md]: this should not be duplicated

File: docs/specs/jobs_specification.md [hunk 3]
```diff
@@ -12,28 +12,32 @@ title: "Specifications | Jobs Specification"
...
 
-Examples of job definition files are available in the [jobs_fixtures directory](https://github.com/monax/cli/tree/master/tests/jobs_fixtures).
+When combined with a package manager to install and resolve dependencies, one can create scripts on top of their smart contracts to coordinate and calculate how certain contracts will interact with each other and evaluate how they will interact through development externalities in real time.
 
-Each job will perform its required action and then it will save the result of its job in a variable which can be utilized by jobs later in the sequence using jobs' [variable specification](/docs/specs/variable_specification).
+Examples of monax job definition files are available in the [jobs_fixtures](/tests/jobs_fixtures) directory.
```
Comment (zramsay) [inline docs/specs/jobs_specification.md]: full URL `https://github.com/monax/cli/blob/master/tests/jobs_fixtures`

File: docs/specs/jobs_specification.md [hunk 4]
```diff
@@ -12,28 +12,32 @@ title: "Specifications | Jobs Specification"
...
+
+Each job will perform its required action and then it will save the result of its job in a variable which can be utilized by jobs later in the sequence using monax:jobs' [variable specification](/docs/specs/variable_specification).
+
+By default, `monax pkgs do` will perform the entire sequence of jobs which has been outlined in a given jobs file.
 
 ## Job Types
 
-Jobs are performed as sequentially based on the order they are given in the jobs definition file. By default, `monax pkgs do` will perform the entire sequence of jobs which has been outlined in a given jobs file.
+Jobs are performed as sequentially based on the order they are given in the jobs definition file. By default monax:jobs will perform the entire sequence of jobs which has been outlined in a given jobs file.
```
Comment (zramsay) [inline docs/specs/jobs_specification.md]: lets not use this notation; its ugly and uninformative and i've purged it from the docs

File: docs/specs/jobs_specification.md [hunk 5]
```diff
@@ -12,28 +12,32 @@ title: "Specifications | Jobs Specification"
...
-* [transaction jobs](#transaction-jobs)
 * [contracts jobs](#contracts-jobs)
+* [transaction jobs](#transaction-jobs)
 * [test jobs](#test-jobs)
 * [other jobs](#other-jobs)
 
 // TODO clean this up (RJ)
 
-For each job which is specified, EPM will parse the following information:
+For each job which is specified, the CLI will parse the following information:
```
Comment (zramsay) [inline docs/specs/jobs_specification.md]: s/CLI/package manager (invoked by `monax pkgs do`)/g

File: docs/specs/jobs_specification.md [hunk 6]
```diff
@@ -70,23 +71,6 @@ The permission job will parse the following information:
 
 {{ insert_definition "jobs.go" "Permission" }}
```
Comment (zramsay) [inline docs/specs/jobs_specification.md]: this needs to go

File: docs/specs/jobs_specification.md [hunk 7]
```diff
@@ -167,4 +151,10 @@ The set job will parse the following information:
 {{ insert_definition "jobs.go" "Set" }}
```
Comment (zramsay) [inline docs/specs/jobs_specification.md]: this needs to go too

File: docs/specs/jobs_specification.md (hunk 4 above)
Comment (VoR0220) [inline docs/specs/jobs_specification.md]: ...but that's what it is. I quite like it. Do you have a better recommendation?

Comment (zramsay) [inline docs/specs/jobs_specification.md]: By default, the package manager (invoked by `monax pkgs do`) will perform...

File: docs/specs/index.md [hunk 8]
```diff
@@ -35,24 +29,18 @@ Service definition files are formatted using `toml`.
 
 ### Jobs Specification
 
+Jobs are a portion of the upcoming Monax package management tooling. They enable ease of automation of smart contract deployment and runtime configuration with special utilizations for Burrow, the Monax blockchain. All tasks can be run through a simple yaml file and cover a range of cases from interaction with smart contracts, assertion testing, name registry, querying the state of the chain, setting permissions on certain addresses, or sending transactions. 
+
```
Comment (zramsay) [inline docs/specs/index.md]: this is duplicated in the jobs_definition file, remove

File: docs/specs/index.md (hunk 1 above)
Comment (zramsay) [inline docs/specs/index.md]: this needs to be fixed still

File: docs/specs/jobs_specification.md (hunk 4 above)
Comment (zramsay) [inline docs/specs/jobs_specification.md]: don't use the `monax:jobs` notation

File: docs/specs/jobs_specification.md [hunk 9]
```diff
@@ -12,104 +12,188 @@ title: "Specifications | Jobs Specification"
...
+
+By default, `monax pkgs do` will perform the entire sequence of jobs which has been outlined in a given jobs file.
 
 ## Job Types
 
-Jobs are performed as sequentially based on the order they are given in the jobs definition file. By default, `monax pkgs do` will perform the entire sequence of jobs which has been outlined in a given jobs file.
+Jobs are performed as sequentially based on the order they are given in the jobs definition file. By default monax:jobs will perform the entire sequence of jobs which has been outlined in a given jobs file.
 
-Job categories are categorize into:
+The CLI will look for an `epm.yaml` file in the current directory unless specified with the `-f` flag. Once it has that, it will from there marshal the values in the file into a job definition which when filled direct monax:jobs on how to execute a task.
```
Comment (zramsay) [inline docs/specs/jobs_specification.md]: think of something better than `monax:jobs`

File: docs/specs/jobs_specification.md [hunk 10]
```diff
@@ -12,104 +12,188 @@ title: "Specifications | Jobs Specification"
...
 
-Examples of job definition files are available in the [jobs_fixtures directory](https://github.com/monax/cli/tree/master/tests/jobs_fixtures).
+When combined with a package manager to install and resolve dependencies, one can create scripts on top of their smart contracts to coordinate and calculate how certain contracts will interact with each other and evaluate how they will interact through development externalities in real time. Utilizing `Monax:Jobs` one can make smart contracts as easy as shell scripting.
```
Comment (zramsay) [inline docs/specs/jobs_specification.md]: the job manager maybe?

File: docs/specs/jobs_specification.md [hunk 11]
```diff
@@ -12,104 +12,188 @@ title: "Specifications | Jobs Specification"
...
 
-Each job will perform its required action and then it will save the result of its job in a variable which can be utilized by jobs later in the sequence using jobs' [variable specification](/docs/specs/variable_specification).
+Examples of monax job definition files are available in the [jobs_fixtures](/tests/jobs_fixtures) directory.
```
Comment (zramsay) [inline docs/specs/jobs_specification.md]: use the full URL: https://github.com/monax/cli/blob/master/tests/....

File: docs/specs/index.md (hunk 8 above)
Comment (VoR0220) [inline docs/specs/index.md]: Hmmm...but what if they're directly linked?

File: docs/specs/jobs_specification.md (hunk 10 above)
Comment (VoR0220) [inline docs/specs/jobs_specification.md]: The JobRunner is what I have it as for the interfaces setup. Would that work better?

File: docs/specs/index.md (hunk 8 above)
Comment (zramsay) [inline docs/specs/index.md]: don't know what you mean? AFAICT, right now it's extra verbosity on the index page. not necessary and already written. there's no sense repeating a paragraph

File: docs/specs/jobs_specification.md (hunk 10 above)
Comment (zramsay) [inline docs/specs/jobs_specification.md]: ++ yes

**Pull request merged**